.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: PYBASEBALL-BUG-
+labels: ''
+assignees: ''
```

Comment:
I'd remove assignees. If things get assigned, I think that should be done by maintainers or whomever wants to do it, I'd rather not have the submitter choose who.